server.go

```diff
@@ -1328,10 +1568,16 @@ type MessagingClient interface {
 	Publish(m *messaging.Message) (index uint64, err error)
 
 	// Creates a new replica with a given ID on the broker.
-	CreateReplica(id uint64) error
+	CreateReplica(replicaID uint64) error
```

Comment:
If we start running into issues then we can re-evaluate but so far it hasn't been an issue. Especially since we have quite a bit of test coverage.